rewards/classes/TreeManager.py

```diff
@@ -204,19 +201,7 @@ def last_propose_start_block(self) -> int:
         return self.badger_tree.lastProposeStartBlock().call()
 
     def get_tx_options(self, account: Account) -> dict:
```

Comment:
This is redundant I guess? Why just not call `create_tx_options(account.address, self.w3, self.chain)` directly ?